plugins/google-analytics-dashboard/src/components/PageLoad/PageLoad.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC, useContext } from 'react';
+import { Typography, Grid, makeStyles } from '@material-ui/core';
+import { Alert, Skeleton } from '@material-ui/lab';
+import api from 'api';
+import { useAsync } from 'react-use';
+import { Context } from 'contexts/Context';
+
+const useStyles = makeStyles({
+  gridItem: {
+    padding: 16,
```

Comment:
`theme.spacing(2)`

but shouldn't the surrounding grid container have a spacing prop set instead?